ironic_bug_dashboard/configs/ironic.json

```diff
@@ -0,0 +1,83 @@
+{
+    "projects": [
+        "bifrost",
+        "ironic",
+        "nova",
+        "ironic-inspector",
+        "ironic-lib",
+        "ironic-prometheus-exporter",
+        "ironic-python-agent",
+        "ironic-python-agent-builder",
+        "ironic-ui",
+        "metalsmith",
+        "networking-baremetal",
+        "python-ironic-inspector-client",
+        "python-ironicclient",
+        "sushy",
+        "sushy-tools",
+        "virtualbmc",
+        "virtualpdu"
+    ],
+    "tagged_projects": [
+        {
+            "project_name": "nova",
+            "tags": "ironic"
+        }
+    ],
+    "priority_required_statuses": ["New", "In Progress", "Triaged", "Confirmed"],
+    "status_priorities": {
+        "In Progress": -10,
+        "Triaged": -5,
+        "Confirmed": -5
+    },
+    "useful_links": [
+        {
+          "url": "https://docs.openstack.org/ironic/latest/index.html",
+          "title": "ironic 24.0.1.dev16"
```

Comment:
let's just use something like "current ironic development version" as title here
the dev version can change very frequently

Reply:
Yea. Makes sense.